### PR TITLE
PMM-6389 added victoriametrics location

### PR DESCRIPTION
**Pull request by f41gh7**

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -114,6 +114,13 @@
       proxy_read_timeout 600;
     }
 
+    # VictoriaMetrics
+
```
Comment (AlekSi) [inline nginx.conf:118]: Please remove that empty line

File: nginx.conf [hunk 2]
```diff
@@ -114,6 +114,13 @@
...
+    location /victoriametrics/ {
```
Comment (AlekSi) [inline nginx.conf:119]: Please remove trailing slash there and below. That will trigger a different behaviour in nginx. That's why we use it in Prometheus above and in Alertmanager below.

But please also test it :)

**Review by AlekSi — CHANGES_REQUESTED** [review]
Please also add it to https://github.com/Percona-Lab/pmm-submodules/pull/1143

File: nginx.conf (hunk 2 above)
Comment (f41gh7) [reply nginx.conf:118]: `vmdb` started without path prefix, so this configuration will route request correctly ( root path at `vmdb` is not supported). Later we will add prometheus path prefix.

Comment (AlekSi) [reply nginx.conf:118]: Will configuration without trailing slashes route requests correctly? If yes, we should use it for consistency

Comment (f41gh7) [reply nginx.conf:118]: Nope, configuration without trailing slashes acts differently. 

 http://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_pass

File: nginx.conf (hunk 1 above)
Comment (f41gh7) [reply nginx.conf]: removed

**Review by AlekSi — APPROVED** [review]

**Pull request merged**